app/views/layouts/application.html.erb

```diff
@@ -1,27 +1,5 @@
 <%= content_for(:content) do %>
-  <div id="main">
-    <%= content_tag('div', flash[:error],   :class => 'flash hide error')   if flash[:error] %>
-    <%= content_tag('div', flash[:warning], :class => 'flash hide warning') if flash[:warning] %>
-    <%= content_tag('div', flash[:notice],  :class => 'flash hide notice')  if flash[:notice] %>
-    <div id="content" class="container">
-      <% unless @page_header.blank? %>
-        <div class="row form-group">
-          <h1 class="col-md-8"><%=h @page_header %></h1>
-        </div>
-      <% end %>
-      <div class="row">
-        <div class="title_filter <%= searchable? ? "col-md-6" : "col-md-4" %>">
-          <%= render "common/searchbar" if searchable? %>
-          <%= yield(:search_bar) %>&nbsp;
-        </div>
-        <div id="title_action" class="<%= searchable? ? "col-md-6" : "col-md-8" %>">
-          <div class="btn-toolbar pull-right"><%=h yield(:title_actions) %></div>
-        </div>
-      </div>
-
-      <%= yield %>
-    </div>
-  </div>
+  <%= render :partial => 'layouts/application_content' %>
 <% end %>
 
 <%= render :template => 'layouts/base' %>
```

Comment:
Can't you use deface and:

```ruby
Deface::Override.new(
  :virtual_path => 'layouts/application',
  :name => 'bastion_assets',
  :insert_after => 'erb[loud]:contains("layouts/base")',
  :text => "<%= render :file => 'bastion/layouts/assets' %>")
```

? I just tried and it loads the assets just fine. It can be done before the `content_for` too if you need that.